blog: let's fully fund Stas (#236)
ddev/ddev.com: src/content/blog/lets-fund-stas-maintainer.md

---
title: "Let's Fully Fund Maintainer Stas"
pubDate: 2024-07-31
# modifiedDate: 2024-07-23

> summary: "Let's fully fund DDEv maintainer Stas Zhuk"

Reviewer: ```suggestion
summary: "Let's fully fund DDEV maintainer Stas Zhuk"
```

author: Randy Fay
featureImage:
  src: /img/blog/2023/10/stas-bench-cropped.jpg
  alt: Stas Zhuk, DDEV Maintainer
categories:
  - Announcements
---

We all want DDEV to be fully maintained at the level you depend on. Now is the time to fully fund maintainer Stas Zhuk so that he does not have to take on other client work. 

> **DDEV's Funding**: DDEV is funded a variety of ways. [Platform.sh covers Randy's salary](platform-sh-becomes-a-lead-sponsor-of-ddev.md), thanks! Individuals and organization sponsors fund DDEV via [GitHub Sponsors](https://github.com/sponsors/ddev) and invoiced support commitments. As the pace of required maintenance and features has increased, we've explained how we intend to grow our maintainership for the long-term in a [couple](recruiting-maintainers.md) of [blog posts](expanding-ddev-maintainer-team.md). (DDEV's "fiscal entity" for funding is the US 501(c)(3) [DDEV Foundation](/foundation)).

Reviewer 2: ```suggestion
**DDEV's Funding**: DDEV is funded in a variety of ways. [Platform.sh covers Randy's salary](platform-sh-becomes-a-lead-sponsor-of-ddev.md), thanks! Individuals and organization sponsors fund DDEV via [GitHub Sponsors](https://github.com/sponsors/ddev) and invoiced support commitments. As the pace of required maintenance and features has increased, we've explained how we intend to grow our maintainership for the long-term in a [couple](recruiting-maintainers.md) of [blog posts](expanding-ddev-maintainer-team.md). (DDEV's "fiscal entity" for funding is the US 501(c)(3) [DDEV Foundation](/foundation)).
```

> **Stas' Role as DDEV Maintainer**: In November, 2023, we [introduced Stas to the community](introducing-maintainer-stas.md) as our second maintainer. Since then, he's done amazing things, working as a part-time paid maintainer. He knows how all the testing and other infrastructure works, has mastered the Golang and Docker codebase, and has contributed hundreds of bugfixes, features, and documentation improvements. As of today, he has contributed [211 commits](https://github.com/ddev/ddev/graphs/contributors) to the master branch of the `ddev/ddev` project alone, second only through the whole history of DDEV to yours truly. Take a look at his [600+](https://github.com/stasadev?tab=overview&from=2024-07-01&to=2024-07-31&org=ddev) contributions to the DDEV org **just in 2024**. He maintains add-ons, supports new contributors, answers questions in Discord, Slack, the issue queue, and Stack Overflow.

Reviewer 2: ```suggestion
**Stas' Role as DDEV Maintainer**: In November, 2023, we [introduced Stas to the community](introducing-maintainer-stas.md) as our second maintainer. Since then, he has done amazing things, working as a part-time paid maintainer. He knows how all the testing and the various infrastructures work, has mastered the Golang and Docker codebase, and has contributed hundreds of bugfixes, features, and documentation improvements. As of today, he has contributed [211 commits](https://github.com/ddev/ddev/graphs/contributors) to the master branch of the `ddev/ddev` project alone, second only through the whole history of DDEV to yours truly. Take a look at his [600+](https://github.com/stasadev?tab=overview&from=2024-07-01&to=2024-07-31&org=ddev) contributions to the DDEV org **just in 2024 alone**. He maintains add-ons, supports new contributors, answers questions in Discord, Slack, Stack Overflow, and the GitHub issue queue.
```
i would avoid the contraction in that case. it is clear that has is used here but with "he's" it could also be "he is" . from a plain language perspective that puts a toll onto the reader.

Reviewer: I replaced `November` with `October` in this suggestion.

> **Why is DDEV Maintenance Important?** You and DDEV are in engulfed in a maelstrom of change. Upstream technologies change on you weekly. Things break from all directions. If DDEV weren't cared for daily, it would fail to serve you within a pretty short time. 

Reviewer: The sentence seemed strange to me, I asked ChatGPT about it:
> I've removed the extra "in" after "are," as it is unnecessary. The word "engulfed" already implies being surrounded or enveloped.

```suggestion
**Why is DDEV Maintenance Important?** You and DDEV are engulfed in a maelstrom of change. Upstream technologies change on you weekly. Things break from all directions. If DDEV weren't cared for daily, it would fail to serve you within a pretty short time. 
```

> **Why is DDEV Maintenance Important?** You and DDEV are in engulfed in a maelstrom of change. Upstream technologies change on your weekly. Things break from all directions. If DDEV weren't cared for daily, it would fail to serve you within a pretty short time. 

Reviewer 2: ```suggestion
**Why is DDEV Maintenance Important?** You and DDEV are in engulfed in a maelstrom of change. Upstream technologies change on your weekly. Things break from all directions. If DDEV weren't cared for on a daily basis, it would fail to serve you within a pretty short period of time. 
```

I'm just naming a tiny few of the things DDEV has had to react to in the last few months.

**What is our current financial situation?** Currently DDEV receives donations in the range of USD$3000-$5000/month. Our expenses are in that same range, the bulk being part-time hourly support for Stas. As of today we have about $18,000 in the bank. Financial reports are provided with every meeting of the [DDEV Advisory Group](https://github.com/orgs/ddev/discussions/categories/ddev-advisory-group).

**What do we want to do? Increase income by about $3000/month.** In order for Stas to be able to leave behind his client work, we need approximately $3000/month more in committed support for the next year. This could be $1,000/month from three organizations (joining fabulous supporting agency [Tag1 Consulting](https://tag1consulting.com)), $500/month from 6 organizations, joining [i-gelb](https://i-gelb.net/). Or $100/month from 30 organizations, joining [so many individual and corporate sponsors](https://ddev.com/#supporters) who keep DDEV going.

> **How can you or your organization support DDEV?** For agencies, hosting folks, other organizations, we're happy to bill you monthly or annually, just ask! It's easy. Some orgs prefer to use GitHub Sponsors, that's fine too. For individuals, the easiest way is [GitHub Sponsors](https://github.com/sponsors/ddev)

Reviewer: ```suggestion
**How can you or your organization support DDEV?** For agencies, hosting folks, other organizations, we're happy to bill you monthly or annually, just ask! It's easy. Some orgs prefer to use GitHub Sponsors, that's fine too. For individuals, the easiest way is [GitHub Sponsors](https://github.com/sponsors/ddev).
```

**Is your organization budgeting for 2025?** Please remember DDEV!

**Get in touch!** I'd be happy to talk to you or your organization. Send [an email](mailto:randy.fay%40ddev.com). Make an [appointment for a video call](https://cal.com/randyfay/30min).

> We're so glad to be bringing you DDEV and working with an amazing community to keep things running! Thanks for all your contributions and for joining us in this journey!

Author: ```suggestion
We're delighted to collaborate with this amazing community on a sustainable and reliable DDEV, and we need your help! Thanks for all your contributions and for joining us in this journey!
```

> We're so glad to be bringing you DDEV and working with an amazing community to keep things running! Thanks for all your contributions and for joining us in this journey!

Reviewer 2: "to be bringing you DDEV" sounds off somehow. it is the correct form but still it sounds strange. would something like "We're so glad to provide DDEV to you and working with an amazing community to keep things running!" ?
